templates/app/offline-worker.js

```diff
@@ -0,0 +1,129 @@
+/* Any copyright is dedicated to the Public Domain.
+ * http://creativecommons.org/publicdomain/zero/1.0/ */
+
+<% importScripts.forEach(function (filepath) {
+%>importScripts('<%- filepath %>');
+<% }); %>
+(function (self) {
+  'use strict';
+
+  // On install, cache resources and skip waiting so the worker won't
+  // wait for clients to be closed before becoming active.
+  self.addEventListener('install', function (event) {
+    event.waitUntil(oghliner.cacheResources().then(function () {
+      if (typeof self.skipWaiting === 'function') {
```

Comment:
Are there browsers that shipped without `skipWaiting`? If there aren't, then we can clean up this function by removing this.